### ISSUE TITLE
Need typescript support 

### ISSUE THREAD

**Issue by Shahaed** [AuthorizeNet/sdk-node#82]
This sdk should be typed like any other modern node package. Not having this feature will force our team switch to Stripe or some other service.

**Comment by pi-guy-in-the-sky** [comment]
I agree with this. I'm working on a team that's using authorize.net for a relatively large project and would strongly benefit from typings. They shouldn't be too hard to generate from the existing JavaScript code. I'm currently considering adding types through the [DefinitelyTyped](https://github.com/DefinitelyTyped/DefinitelyTyped) project, which would then be theoretically installed via `npm install --save-dev @types/authorizenet`, since it seems like this project isn't as actively maintained. I'll update this thread if I make any progress.

**Comment by PatrickEGorman** [comment]
Types were created around a year ago
```npm install --save @types/authorizenet```

https://www.npmjs.com/package/@types/authorizenet

**Comment by DASPRiD** [comment]
Let me shamelessly recommend `better-authorize-net` :)

https://www.npmjs.com/package/better-authorize-net
